### PR TITLE
Adf 129 prototype search results

### PR DESCRIPTION
**Pull request by payekumi**
## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- [Issue 129: Prototype search results](https://palantir.atlassian.net/browse/ADF-129)

**Jira Ticket**
- [ADF-129: Prototype search results](https://palantir.atlassian.net/browse/ADF-129)

## Description
Show different “states” – Member (member badge) / Non-member (no member badge) / Verified (verified check-mark) / Not-verified (no check-mark)

## To Test
Navigate to the Dr-finder search results header and you will see the following 
Screen Shot 2022-10-06 at 11 27 08 AM

## Relevant Screenshots/GIFs
Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.

## Remaining Tasks
Create if statements so that the member box will be displayed 
Create if statement for the check mark verified

## Additional Notes

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/_patterns/02-molecules/dr-finder-search-result-card/dr-finder-search-result-card.twig [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    {% if drFinderSearchResultCard.verified %}
+      {% include '@atoms/media/icons/svg/icon-completed.twig'%}
+    {% endif %}
+  </div>
+  <div class="dr-finder-search-result-card__content">
+    <div class="dr-finder-search-result-card__content__name ama__h2">{{ drFinderSearchResultCard.data.name|raw }}</div>
+    <div class="dr-finder-search-result-card__content__address">{{ drFinderSearchResultCard.data.address }}</div>
+    <div class="dr-finder-search-result-card__content__specialty">{{ drFinderSearchResultCard.data.specialty|raw }}</div>
+    {% if drFinderSearchResultCard.member %}
+      {% include '@atoms/dr-finder-badge/dr-finder-badge.twig'%}</div>
```
Comment (nstriedinger) [inline styleguide/source/_patterns/02-molecules/dr-finder-search-result-card/dr-finder-search-result-card.twig]: @payekumi Looks like you're missing a `<div>` somewhere

File: styleguide/source/assets/scss/02-molecules/_dr-finder-search-result-card.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
+.dr-finder-search-result-card{
+  display: flex;
+  padding: $gutter;
+  margin-bottom: $gutter;
+  border: solid 1px;
+  box-shadow: 0px 1px 4px #4d4d4d;
+  &:hover{background-color: #f7f3fd;}
```
Comment (nstriedinger) [inline styleguide/source/assets/scss/02-molecules/_dr-finder-search-result-card.scss]: Odd spacing -- be consistent.

Comment (nstriedinger) [reply styleguide/source/assets/scss/02-molecules/_dr-finder-search-result-card.scss]: use color `vars` for hex colors (in `_colors.scss`)

File: styleguide/source/assets/scss/02-molecules/_dr-finder-search-result-card.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
+.dr-finder-search-result-card{
+  display: flex;
+  padding: $gutter;
+  margin-bottom: $gutter;
+  border: solid 1px;
+  box-shadow: 0px 1px 4px #4d4d4d;
```
Comment (nstriedinger) [inline styleguide/source/assets/scss/02-molecules/_dr-finder-search-result-card.scss]: Unit of measure 'px' is redundant -- just use `0`

Comment (payekumi) [reply styleguide/source/assets/scss/02-molecules/_dr-finder-search-result-card.scss]: I have removed the redundant px

File: styleguide/source/assets/scss/02-molecules/_dr-finder-search-result-card.scss (hunk 2 above)
Comment (payekumi) [reply styleguide/source/assets/scss/02-molecules/_dr-finder-search-result-card.scss]: I have made the change to use the vars instead of the hex

**Pull request merged**